README.md

```diff
@@ -57,17 +57,17 @@ module.exports = function(el) {
 };
 ```
 
-Now whenever you add an element with `data-wt="popup"` to the DOM, a click listener will be bound:
+Now whenever you add an element with `data-wt="alert"` to the DOM, a click listener will be bound:
 
 ```
-var content = $.get(someUrl); // Contains a "popup" block
-$('body').append(content); // "popup" block is initialized automatically
+var content = $.get(someUrl); // Contains an "alert" block
```

Comment:
might be worth mentioning that Regulator recognizes this because the `content` contains an element with  `data-wt="alert"`. It's implied above, but the README is for teaching, and repetition is for teaching, and repetition is for teaching